src/test/java/LoginTests.java

```diff
@@ -6,31 +6,33 @@
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-public class  LoginTests extends BaseTest1 {
-   private String invalidEmail="holost@gmail.com";
-   private By avatarLocator=By.xpath("//*[contains(@alt,'Avatar of')]");
+public class LoginTests extends BaseTest {
+    protected String myEmail = "holostenco.yuliya@gmail.com";
+    protected String myPassword = "te$t$tudent";
+    private String invalidEmail = "holost@gmail.com";
 
-   @Test
-    public void LoginValidEmailValidPasswordTest(){
-        LoginPage loginPage = new LoginPage(driver);
-        HomePage homePage= new HomePage(driver);
+    @Test
+    public void LoginValidEmailValidPasswordTest() {
+        LoginPage loginPage = new LoginPage(getDriver());
+        HomePage homePage = new HomePage(getDriver());
         loginPage.login();
-        WebElement avatarIcon= driver.findElement(avatarLocator);
-        Assert.assertTrue(avatarIcon.isDisplayed());
+        homePage.getAvatarIcon();
+        homePage.userAvatarIsDisplayed();
+        Assert.assertTrue(homePage.userAvatarIsDisplayed());
```

Comment:
**Don't repeat yourself!** (  In Java, it means don't write the same code repeatedly. )

```
   @Test
    public void LoginValidEmailValidPasswordTest() {
        LoginPage loginPage = new LoginPage(getDriver());
        HomePage homePage = new HomePage(getDriver());
        loginPage.login();
        Assert.assertTrue(homePage.userAvatarIsDisplayed());
    }
```